contrib/verify-commits/verify-commits.py

```diff
@@ -150,7 +150,20 @@ def main():
             current_tree = subprocess.check_output([GIT, 'show', '--format=%T', current_commit]).decode('utf8').splitlines()[0]
             subprocess.call([GIT, 'checkout', '--force', '--quiet', parents[0]])
             subprocess.call([GIT, 'merge', '--no-ff', '--quiet', '--no-gpg-sign', parents[1]], stdout=subprocess.DEVNULL)
-            recreated_tree = subprocess.check_output([GIT, 'show', '--format=format:%T', 'HEAD']).decode('utf8').splitlines()[0]
+
+            # This merge-tree functionality requires git >= 2.38. The
```

Comment:
I think, bitcoin/bitcoin#27058 should be done firstly; because this backport supposed to add only some try-except but doesn't replace `show --format` to `merge-tree`